test/parallel/test-exec-max-buffer.js

```diff
@@ -9,3 +9,14 @@ exec(cmd, { maxBuffer: 5 }, function(err, stdout, stderr) {
   assert.ok(err);
   assert.ok(/maxBuffer/.test(err.message));
 });
+
+if (!common.isWindows) {
```

Comment:
the spawn execute echo with unicode has unexpected behavior.

发自我的 iPhone

> 在 2015年7月7日，上午1:11，Jeremiah Senkpiel <notifications@github.com> 写道：
> 
> In test/parallel/test-exec-max-buffer.js:
> 
> > @@ -9,3 +9,14 @@ exec(cmd, { maxBuffer: 5 }, function(err, stdout, stderr) {
> >    assert.ok(err);
> >    assert.ok(/maxBuffer/.test(err.message));
> >  });
> > +
> > +if (!common.isWindows) {
> Is this because echo doesn't exist on windows? Is there anything we can do to test windows?
> 
> I'm not terribly comfortable with this kind of testing. :s
> 
> —
> Reply to this email directly or view it on GitHub.
>